packages/http/fetch/package.json

```diff
@@ -44,7 +44,7 @@
         "test:es": "tsc -b tsconfig.es.test.json && mocha 'dist/es/test/common/**/*.js'  --require  esm && mocha 'dist/es/test/node/**/*.js'  --require  esm"
     },
     "dependencies": {
-        "@microsoft/kiota-abstractions": "1.0.0-preview.4",
+        "@microsoft/kiota-abstractions": "^1.0.0-preview.5",
```

Comment:
```suggestion
        "@microsoft/kiota-abstractions": "1.0.0-preview.5",
```

Reply:
The preview version will be locked to a specific version of abstractions. 

The changes in a higher version of abstractions may/may not be compatible.